package.json

```diff
@@ -11,18 +11,18 @@
   },
```

Comment:
Is there a specific reason that we would use "^" instead of "~" :-)? Aren't we risking that the packages minor number updates could result in some breaking changes that will result in the code could not work? At least the risk is higher if using ^ instead of ~ :-)

Reply:
While kinda true, as long a the major version is above 1 minor updates should not be breaking